docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -103,7 +103,7 @@ the background.
 ### Setting the cascading deletion policy
 
 To control the cascading deletion policy, set the `deleteOptions.propagationPolicy`
-field on your owner object. Possible values include "Orphan",
+field on the `deleteOptions` argument when deleting an Object. Possible values include "Orphan",
```

Comment:
Line 105 has already fully spelled out the field name.
The revision makes it more confusing ... am I supposed to set `deleteOptions.deleteOptions.propagationPolicy`?
One workaround would be change the field name to `propagationPolicy` for clarity.

Reply:
Good catch - I totally agree.

Changed! PTAL.